releases/release-6.1.1.md

```diff
@@ -0,0 +1,170 @@
+---
+title: TiDB 6.1.1 Release Notes
+---
+
+# TiDB 6.1.1 Release Notes
+
+Release date: 2022-xx-xx
+
+TiDB version: 6.1.1
+
+## Compatibility changes
+
++ TiDB
+
+    (dup: release-6.2.0.md > Bug fixes> TiDB)- Fix the issue that `SHOW DATABASES LIKE …` is case-sensitive [#34766](https://github.com/pingcap/tidb/issues/34766)
+    - Change the default value of [`tidb_enable_outer_join_reorder`](/system-variables.md#tidb_enable_outer_join_reorder-new-in-v610) from `1` to `0`, which disables Join Reorder's support for Outer Join is enabled by default.
+
+## Improvements
+
+- Add some contents in the `TiDB-community-toolkit` binary package. For details, see [TiDB Installation Packages](/binary-package.md).
+- Add a document to introduce TiDB's support for different operating systems. See [].
+
++ TiDB
+
+<!-- <planner> -->
+(dup: release-6.2.0.md > # Performance)[User document](/optimizer-hints.md#semi_join_rewrite) [#35323](https://github.com/pingcap/tidb/issues/35323)
+(dup: release-5.2.4.md > Bug fixes> TiDB)- Fix the issue that partitioned tables cannot fully use indexes to scan data in some cases [#33966](https://github.com/pingcap/tidb/issues/33966)
+
+<!-- <transaction> -->
+(dup: release-6.2.0.md > Bug fixes> TiDB)- Avoid sending requests to unhealthy TiKV nodes to improve availability [#34906](https://github.com/pingcap/tidb/issues/34906)
+
++ TiKV
+
+    (dup: release-6.2.0.md > Improvements> TiKV)- Support compressing the metrics response using gzip to reduce the HTTP body size [#12355](https://github.com/tikv/tikv/issues/12355)
+    - Support filter useless metrics samples to reduce the metrics data size. [#12355](https://github.com/tikv/tikv/issues/12355)
+    (dup: release-6.2.0.md > Improvements> TiKV)- Support dynamically modifying the number of sub-compaction operations performed concurrently in RocksDB (`rocksdb.max-sub-compactions`) [#13145](https://github.com/tikv/tikv/issues/13145)
+
```

Comment:
PD
 - Improve the speed of the balance region during the stage `close to finish` #5320